LSPS2/README.md

```diff
@@ -0,0 +1,864 @@
+# LSPS2 JIT Channel Negotiation
+
+|Name    | `jit_channels`           |
+|--------|--------------------------|
+|Version | 1                        |
+|Status  | For Implementation       |
+
+## Motivation
+
+A "JIT Channel" is a channel opened in response to an incoming payment
+from the public network to a client, via the LSP.
+This allows a client with no Lightning channels to start receiving on
+Lightning, and have the cost of their inbound liquidity be deducted from their
+first received payment.
+
+### Trust Models
+
+As of this version, the trust model is that the LSP trusts clients to actually
+claim their payment once the channel is opened and the payment is forwarded.
```

Comment:
I havent seen anything mentioned when the LSP can switch to "client-trusts-LSP" or how this is technically handled. I am aware we still need to work out the rationale behind this, but I am also missing the technical way of how to switch.

Reply:
This is done by using LSPS3 and reporting a `conditions_apply` error in the `lsps3.getpromise` interface.

Reply:
As discussed in the telegram group. This spec should be independent of LSPS3. So I would appreciate to have this text in here.

Reply:
There's now a separate flag for forcing client to trust LSP.